man/man1/clean_output.1

```diff
@@ -0,0 +1,14 @@
+.\"  Man page for clean_output
+.TH man (1) "11 November 2020" "1.0" "clean_output  man page"
+.SH NAME
+clean_output  \- Remove output files
+.SH SYNOPSIS
+clean_output
+.SH DESCRIPTION
+Remove any output files generated by viz_pipe using the "rm -rf" and "rm f" commands
```

Comment:
We should specify what types of output files. Specifically any files with config*.dot and *.dot.png